backend/tarsy/services/dashboard_broadcaster.py

```
@@ -276,75 +64,43 @@ async def broadcast_message(
-            
-            batch = self.active_batches[channel]
-            if batch.add_message(message):
-                # Batch is ready, send immediately
-                await self._send_batch(channel, batch)
-                del self.active_batches[channel]
-            
-            # Record messages for throttling
-            for user_id in eligible_users:
+        # Send immediately to all eligible users
+        sent_count = 0
+        message_dict = message.model_dump() if hasattr(message, 'model_dump') else message
```

Ruff (0.12.2)

Line too long (90 > 88)

(E501)